automerge-frontend/src/change_context.rs

```diff
@@ -123,7 +123,7 @@ impl<'a> ChangeContext<'a> {
     /// change is complete we can `commit` the change context.
     ///
     pub fn apply_diff(&mut self, diff: &Diff) -> Result<(), InvalidPatch> {
-        println!("The diff: {:?}", diff);
+        // println!("The diff: {:?}", diff);
```

Comment:
Had to comment this line out since it ended up in the changes file when importing. 😅

Reply:
Ah, that definitely shouldn't be there, I must have accidentally left it in whilst debugging. Please feel free to remove it.

Reply:
⚡ https://github.com/automerge/automerge-rs/pull/21/commits/5e2d74d13e7166c22950e4520da00f32bc76d01c